### PR TITLE
Menu Process Done

### PR DESCRIPTION
**Pull request by GedePrama**
Check this backlog:
- Menu Process
- Text to Voice Translation
- Replay Button Logic

### CODE REVIEW

File: Satuju/Helper/ViewHasScrollEnabled.swift [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import SwiftUI
+
+extension View {
+    func hasScrollEnabled(_ value: Bool) -> some View {
+        self.onAppear {
+            UITableView.appearance().isScrollEnabled = value
+        }
+    }
+}
```
Comment (mrifkiw) [inline Satuju/Helper/ViewHasScrollEnabled.swift:16]: penamaan helpernya udah bener gak nih can @yrojx ?

Comment (yrojx) [reply Satuju/Helper/ViewHasScrollEnabled.swift:16]: View+hasScrollEnabled

File: Satuju/Helper/ViewHasScrollEnabled.swift [hunk 2]
```diff
@@ -0,0 +1,16 @@
+//
+//  ViewHasScrollEnabled.swift
```
Comment (yrojx) [inline Satuju/Helper/ViewHasScrollEnabled.swift:2]: View+hasScrollEnabled

**Review by yrojx — CHANGES_REQUESTED** [review]

**Comment by yrojx** [comment]
ntr kalo udah, aku resolve-in konfliknya

**Pull request closed without merging**